packages/emulation/libretro-fbalpha/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libretro-fbalpha"
-PKG_VERSION="9ceab7f6293864591e63a2669396dc200e3246bb"
-PKG_SHA256="26eb3e4c3f5c7238fd9632c055d2941be50348fc16998c774126aba2e81508d2"
+PKG_VERSION="20daa807957d7fce64371620271b502811318163"
```

Comment:
This hash is no longer available upstream.

Reply:
tx fixed